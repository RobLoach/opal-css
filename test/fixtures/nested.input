.phone {
  &_title {
    width: 500px;
  }

  body.is_dark {
    color: white;
  }
  img {
    display: block;
  }
}
