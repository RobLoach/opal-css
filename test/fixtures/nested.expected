.phone_title{width:500px}.phone body.is_dark{color:#fff}.phone img{display:block}
